scripts/pinned_browsers.py

```diff
@@ -48,18 +48,15 @@ def get_chrome_milestone():
     )[-1]
 
 
-def chromedriver(selected_version):
+def chromedriver(selected_version, workspace_prefix=""):
     content = ""
 
     drivers = selected_version["downloads"]["chromedriver"]
 
     linux = [d["url"] for d in drivers if d["platform"] == "linux64"][0]
     sha = calculate_hash(linux)
-
-    content = (
-        content
-        + """    http_archive(
-        name = "linux_chromedriver",
+    content = content + """    http_archive(
```

Comment:
There's a reason everything is formatted the way it is. I don't remember that reason, but I made a point of doing it that way: 163167e30f4168edbd7f4435e98dca87346f2481